backend/controllers/productController.ts

```diff
@@ -7,14 +7,15 @@ export const getProducts = asyncHandler(async (req: any, res: Response) => {
   res.json(products)
 })
 
+export class NotFoundError extends Error {
```

Comment:
Can we outback this in its own file?

Reply:
Yup, nothing changes if I remove "export" because I'm only using it inside it.
I'll remove it. Anything else?